kitsune/kbforums/views.py

```diff
@@ -446,7 +446,6 @@ def locale_discussions(request):
     threads_ = sort_threads(threads, sort, desc)
 
     # Ignore sticky-ness:
```

Comment:
This comment refers to the line which was removed, so should be removed too

Reply:
Removed!